var/spack/repos/builtin/packages/py-fastfold/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyFastfold(PythonPackage):
+    """Optimizing Protein Structure Prediction Model Training and Inference on
+    GPU Clusters."""
+
+    homepage = "https://github.com/hpcaitech/FastFold"
+    url = "https://github.com/hpcaitech/FastFold/archive/refs/tags/0.2.0.tar.gz"
+
+    maintainers = ["meyersbs"]
+
+    version("0.2.0", sha256="6760dbae9809b8b26219c9477489d34325807be504098901d0375fbdc3103f88")
+
+    # From README:
+    depends_on("python@3.8:3.9", type=("build", "run"))
```

Comment:
Do Python 3.10 or 3.11 work? 3.10 is our default version

Reply:
Building now